### PR TITLE
Fix: Use domainNameAlias for the authority in the mediaPlayerEntry when available and improve error logging by the 5GMS AF

### LINKED ISSUE #28: ServiceAccessInformation.streamingAccess.mediaPlayerEntry should use ContentHostingConfiguration.distributionConfigurations.domainNameAlias as the authority name when present.
**Describe the bug**
If the Application Function is configured with a `ContentHostingConfiguration` which specifies `distributionConfigurations.domainNameAlias`, the alias name should be used in the `ServiceAccessInformation.streamingAccess.mediaPlayerEntry` URL as the authority in preference to the `canonicalDomainName`.

**To Reproduce**
1. Configure a `ContentHostingConfiguration` which uses `domainNameAlias` in its distributionConfigurations.
2. Run the Application Function.
3. Fetch the Service Access Information
4. Observe that the `mediaPlayerEntry` URL references the `canonicalDomainName` of the Application Server instead of the provided `domainNameAlias`.

**Expected behavior**
The `mediaPlayerEntry` URL authority should be the same as the `domainNameAlias`.

**Desktop:**
 - OS: Ubuntu 22.04.1
 - Browser: curl

### PR DESCRIPTION
**Pull request by davidjwbbc**
Closes #28 

Also adds a little extra logging around ContentHostingConfiguration reading.

### CODE REVIEW

**Comment by davidjwbbc** [comment]
Will backport to v1.0.x in another PR.

File: src/5gmsaf/provisioning-session.c [hunk 1]
```diff
@@ -82,6 +82,13 @@ msaf_provisioning_session_create(char *provisioning_session_type, char *asp_id,
     msaf_provisioning_session->certificate_map = msaf_certificate_map();
     ogs_hash_set(msaf_self()->provisioningSessions_map, ogs_strdup(msaf_provisioning_session->provisioningSessionId), OGS_HASH_KEY_STRING, msaf_provisioning_session);
 
+#if 1 /* TODO: Remove when content hosting configuration is available via M1 interface */
+    msaf_provisioning_session->contentHostingConfiguration = msaf_content_hosting_configuration_create(msaf_provisioning_session);
+    media_player_entry = media_player_entry_create(msaf_provisioning_session->provisioningSessionId, msaf_provisioning_session->contentHostingConfiguration);
+    ogs_assert(media_player_entry);
```
Comment (rjb1000) [inline src/5gmsaf/provisioning-session.c:88]: The Media Player Entry is an optional property, so I am not sure this assertion is correct.

Comment (davidjwbbc) [reply src/5gmsaf/provisioning-session.c:88]: We always create the mediaPlayerEntry and the assertion is just that the memory for the mediaPlayerEntry structure got allocated. This is how the open5gs code usually catches "Out Of Memory" errors.

Comment (rjb1000) [reply src/5gmsaf/provisioning-session.c:88]: Understood, @davidjwbbc.

But I have to say it does seem a bit weird that you always create the `mediaPlayerEntry` member when this is an optional property of the underlying YAML data type.

**Comment by davidjwbbc** [comment]
This partially fixes #29 too.

**Pull request merged**